MatrixSDK/Space/MXSpaceService.swift

```diff
@@ -77,29 +84,38 @@ public class MXSpaceService: NSObject {
             }
         }
     }
-    private var spacesPerId: [String:MXSpace] = [:]
-    
-    private var isGraphBuilding = false;
-    private var isClosed = false;
     
+    // MARK: Public
+
+    /// The instance of `MXSpaceNotificationCounter` that computes the number of unread messages for each space
     public let notificationCounter: MXSpaceNotificationCounter
     
+    /// List of `MXroomSummary` of the high level spaces.
     public var rootSpaceSummaries: [MXRoomSummary] {
         return self.graph.rootSpaceIds.compactMap { spaceId in
             self.session.roomSummary(withRoomId: spaceId)
         }
     }
     
+    /// List of `MXroomSummary` of all spaces known by the user.
+    public var spaceSummaries: [MXRoomSummary] {
+        return self.graph.spaceRoomIds.compactMap { spaceId in
+            self.session.roomSummary(withRoomId: spaceId)
+        }
+    }
+
+    /// `true` if the `MXspaceService` instance needs to be update (e.g. the instance was busy while `handleSync` was called). `false` otherwise
```

Comment:
```suggestion
    /// `true` if the `MXSpaceService` instance needs to be updated (e.g. the instance was busy while `handleSync` was called). `false` otherwise
```